server/src/operators/analytics_operator.rs

```diff
@@ -1382,6 +1382,49 @@ pub async fn set_query_rating_query(
     Ok(())
 }
 
+pub async fn set_rag_query_rating_query(
+    data: RateQueryRequest,
+    dataset_id: uuid::Uuid,
+    clickhouse_client: &clickhouse::Client,
```

Comment:
Can you add this to openapi spec.

You might also need to regenerate the client

Reply:
Looking at the wrong spot